front-end/packages/robustar/src/views/ImageList.vue

```diff
@@ -105,49 +110,113 @@
+                @click="selectImageSteps++"
+              >
+                Continue
+              </v-btn>
+              <v-btn color="warning" outlined @click="selectImageSteps--"> Back </v-btn>
+              <v-alert v-if="imageEndIdx < imageStartIdx" dense text type="error" class="mt-4">
+                End Index smaller than Start Index
+              </v-alert>
+            </v-stepper-content>
+
+            <v-stepper-step :complete="selectImageSteps > 3" step="3"
+              >Apply selection</v-stepper-step
             >
-              <v-icon class="mr-2">mdi-auto-fix</v-icon> Auto Annotate
-            </v-btn>
-          </div>
-          <v-btn icon color="grey" large @click="showExtraSettings = !showExtraSettings">
-            <v-icon v-if="!showExtraSettings">mdi-chevron-double-right</v-icon>
-            <v-icon v-else>mdi-chevron-double-left</v-icon>
-          </v-btn>
+
+            <v-stepper-content step="3">
+              <v-btn
+                depressed
+                outlined
+                color="primary"
+                @click="
+                  $router.push({
+                    name: 'InfluencePad',
+                    params: { startIdx: imageStartIdx, endIdx: imageEndIdx },
+                  })
+                "
+              >
+                <v-icon class="mr-2">mdi-vector-link</v-icon> Influence
+              </v-btn>
+              <v-btn
+                depressed
+                outlined
+                color="primary"
+                class="mx-4"
+                @click="
+                  $router.push({
+                    name: 'AutoAnnotatePad',
+                    params: { startIdx: imageStartIdx, endIdx: imageEndIdx },
+                  })
+                "
+              >
+                <v-icon class="mr-2">mdi-auto-fix</v-icon> Auto Annotate
+              </v-btn>
+              <v-btn color="warning" outlined @click="selectImageSteps--"> Back </v-btn>
+            </v-stepper-content>
+          </v-stepper>
+          <v-tooltip right>
+            <template v-slot:activator="{ on, attrs }">
+              <v-btn
+                icon
+                color="grey"
+                large
+                @click="showExtraSettings = !showExtraSettings"
+                v-bind="attrs"
+                v-on="on"
+              >
+                <v-icon v-if="!showExtraSettings">mdi-chevron-double-right</v-icon>
+                <v-icon v-else>mdi-chevron-double-left</v-icon>
+              </v-btn>
+            </template>
+            <span>process a batch of selected images</span>
+          </v-tooltip>
         </v-sheet>
+
+        <v-alert
+          v-if="$root.imageSplit !== 'annotated' && showExtraSettings"
+          text
+          type="primary"
+          height="100"
+          class="pa-8 text-h5"
+        >
+          You've selected <strong>{{ imageEndIdx - imageStartIdx + 1 }}</strong> image(s)
```

Comment:
This is very nice! A few thoughts here (not necessarily correct)
- Maybe change the text to `x image(s) selected` to make the text shorter
- We can make the `image(s) selected` smaller, an make the number larger to further save space. A good reference is ant design's `statistic` component here: https://ant.design/components/statistic. 
- Now `imageEndIdx - imageStartIdx + 1` is negative when selecting the first image. Maybe wrap it with a `Max(0, x)`?

Reply:
Thanks for your suggestions! Make sense